global-hooks/openapi/doc-ru-config-values.yaml

```
@@ -116,4 +116,4 @@ properties:
               Proxy URL для HTTPS-запросов.
           noProxy:
             description: |
-              Список IP и доменных имен для которых проксирование не применяется.
+              Список IP и доменных имен для которых проксирование не применяется. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале, например ".example.com".
```

```suggestion
              Список IP и доменных имен, для которых проксирование не применяется. Для всех доменов третьего уровня (wildcard), используйте домен второго, добавив точку в начале, например ".example.com".
```
Perhaps it makes sense to rewrite it as _`Для настройки wildcard-доменов используйте написание вида ".example.com"`_ (see apolovov's suggestion above).